Upgrade prometheus-operator chart

gabeduke opened the pull request:
* Upgrade helm chart from `6.20.3` to `8.2.0` and update the app version in the Civo manifest
* Update CRD's to reflect the new helm chart version
* Add bash loop to check for CRD's to exist before installing the chart

If your pull request concerns an existing Marketplace application, please make sure you have:

* [x] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [x] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [x] Tested that the application works with the proposed updates applied (including a screenshot)

running chart externally with helm3 (rather than install tiller for testing):

```bash
~ % helm3 list
NAME               	NAMESPACE 	REVISION	UPDATED                                	STATUS  	CHART                    	APP VERSION
prometheus-operator	monitoring	1       	2019-11-15 12:56:06.365987947 -0500 EST	deployed	prometheus-operator-8.2.0	0.34.0   
```

----

Updated chart with servicemonitors working will now show actual data in prom/grafana:

![Screenshot from 2019-11-19 21-15-33](https://user-images.githubusercontent.com/4111646/69431690-3f3abb80-0d06-11ea-907e-11e891e8fba9.png)

gabeduke commented:
/assign @akosveres

akosveres commented on prometheus-operator/manifest.yaml:
```diff
@@ -1,7 +1,7 @@
 name: prometheus-operator
 title: "Prometheus Operator"
-version: 0.32.0
+version: 8.2.0
```
This should be the version of the prometheus-operator, not the helm chart.

gabeduke replied:
Oops I misunderstood that. Fixed on the next commit.

akosveres commented:
The app should be upgraded to 0.34.0 based on your proof from above.

gabeduke commented:
Of course :man_facepalming: . Fixed in this commit. 

I also added a check for  the CRDs to exist before installing the chart. The increment will only work in bash so if install script is being called with `sh` there might be problems.

gabeduke commented:
/assign @kaihoffman @andyjeffries

andyjeffries commented:
Thanks for the work on this guys, merged and will be deployed in the next hour or so.
